pkgdb2/api/packages.py

```diff
@@ -1105,9 +1139,9 @@ def api_package_critpath():
     return jsonout
 
 
-@API.route('/package/<package>/monitor/<status>', methods=['POST'])
+@API.route('/package/<namespace>/<package>/monitor/<status>', methods=['POST'])
```

Comment:
Nope, this is a POST only endpoint, I'm in favor of breaking the API there.

What do you think?